warehouse/manage/views.py

```diff
@@ -566,6 +566,7 @@ def default_response(self):
                 user_id=self.request.user.id,
                 macaroon_service=self.macaroon_service,
                 project_names=self.project_names,
+                description="",
```

Comment:
Might be better to do `{{ create_macaroon_form.description.data or ""}}` in the template instead of this.